programs/ssl/ssl_server2.c

```diff
@@ -2359,17 +2372,31 @@ int main( int argc, char *argv[] )
         goto exit;
 #else
 #if defined(MBEDTLS_RSA_C)
+
+#if defined(MBEDTLS_PEM_PARSE_C)
```

Comment:
@k-stachowiak So far, `certs.h` only provided `mbedtls_test_srv_crt`, dispatching between both PEM / DER and between RSA / EC; here, we explicitly want RSA and EC CRTs, and only need the automatic dispatch between PEM and DER, which `certs.c` didn't provide. But you're entirely right that, if we need it, it should be done in `certs.c` and not here. 

I've now adapted `certs.c` to provide `mbedtls_test_srv_crt_{rsa|ec}` as either `mbedtls_test_srv_crt_{rsa|ec}_pem` or `mbedtls_test_srv_crt_{rsa|ec}_der` depending on which is available, so that `ssl_server2.c` can stay unchanged.